src/main.rs

```diff
@@ -57,7 +57,7 @@ enum Commands {
     },
     /// Calculate procurement indicators from OCDS compiled releases in a line-delimited JSON file
     ///
-    /// The result is a JSON object, in which the keys are one of "OCID", "Buyer", "ProcuringEntity"
+    /// The result is a JSON object, in which the keys are one of "OCID", "Buyer", "`ProcuringEntity`"
```

Comment:
@jpmckinney FYI  cargo was complaining about this:

````
error: item in documentation is missing backticks
  --> src/main.rs:60:85
   |
60 |     /// The result is a JSON object, in which the keys are one of "OCID", "Buyer", "ProcuringEntity"
   |                                                                                     ^^^^^^^^^^^^^^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown
   = note: `-D clippy::doc-markdown` implied by `-D clippy::pedantic`
   = help: to override `-D clippy::pedantic` add `#[allow(clippy::doc_markdown)]`
help: try
   |
60 |     /// The result is a JSON object, in which the keys are one of "OCID", "Buyer", "`ProcuringEntity`"
   |                                                                                     ~~~~~~~~~~~~~~~~~

error: could not compile `ocdscardinal` (bin "ocdscardinal") due to 1 previous error

Reply:
Add `-Aclippy::doc-markdown` to `.pre-commit-config.yaml` since this is a false positive.

Reply:
Strangely, I don't get this. We can maybe review what version of cargo/Rust you're using.